README.md

```diff
@@ -121,4 +121,19 @@ vendor/bin/roave-backward-compatibility-check --help
 
 ## Configuration
 
-There are currently no configuration options available.
+The file `.roave-backward-compatibility-check.json` is read from the current working directory (when it exists) and sets configuration for the command.
+
+It's expected to be a JSON encoded file that, optionally, contains the following properties:
```

Comment:
We can implement another format, I just think JSON is the simplest or most predictable...

Reply:
I wonder if XML might be a nicer option, on account of not having to escape the regex?

Reply:
I've implemented this quickly only to put the idea out there, JSON was the quickest thing I could use.

I'll change it to XML and add an XSD schema, my only question is: are you fine with the extra dependencies to be added to composer.json (ext-dom / ext-simplexml / ext-xmlreader / whateva)?

Reply:
I'm fine with either, as long as there's a schema

Reply:
https://github.com/Roave/BackwardCompatibilityCheck/pull/737/commits/a7619cdd4144eb6c6811ce656aa0390f0404068a provides the XML+XSD implementation :+1: